module/surveys/validator/v1.js

```diff
@@ -27,10 +27,6 @@ module.exports = (req) => {
     getDetailsByLink: function () {
       req.checkParams('_id').exists().withMessage('required link');
     },
-
-    details: function () {
```

Comment:
@praveenKDass we don't need this?

Reply:
Yes we do't need this as details can be called by passing surveyId or solution id or through link @VISHNUDAS-tunerlabs